utils/config_models/settings.py

```
@@ -248,6 +248,18 @@ def get_sleep_time(self):
         else:
             raise ValueError("No cooldown?")
 
+    def get_check_time(self):
+        if self.checkTime:
+            if self.checkTime[0] > self.checkTime[1]:
+                raise ValueError("Max cooldown must be greater than min.")
+
+            if self.checkTime[0] == self.checkTime[1]:
+                raise ValueError("Both min and max cooldown same..")
+
+            return random.uniform(self.checkTime[0], self.checkTime[1])
```

Ruff (0.15.21)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)